docs/using-mycroft-ai/basic-commands.md

```diff
@@ -1,104 +1,328 @@
+###
+## The Default **Skills**
+### Alarm - *Manages a collection of alarms.  Weekly repetition is supported.*
+##### Set an alarm
+> Hey Mycroft, wake me in two hours <br>
+> Hey Mycroft, wake me up at 10:23pm <br>
+> Hey Mycroft, set an alarm for two hours <br>
+> Hey Mycroft, set an alarm for 3pm <br>
+> Hey Mycroft, add an alarm for Monday at 8:20 <br>
+> Hey Mycroft, create alarms for every weekday at 6 <br>
+> Hey Mycroft, start a repeating alarm for Fridays at 9:30pm <br>
+> Hey Mycroft, set an alarm *<- - - - - Mycroft asks for time and date*
+##### Snooze the active alarm - *Defaults to 9 minutes*
+> Hey Mycroft, snooze <br>
+> Hey Mycroft, snooze for 15 minutes <br>
+> Hey Mycroft, give me 2 more hours
+##### Delete/stop/cancel/dismiss/remove 
+> Hey Mycroft, stop the alarm <br>
+> Hey Mycroft, delete the Tuesday alarm for 9 <br>
+> Hey Mycroft, remove all alarms
+##### Query alarms
+> Hey Mycroft, when is the next alarm? <br>
+> Hey Mycroft, what are the alarms for today?
+##### Change alarm sound
+> Hey Mycroft, change the sound of the alarm
+###
+### Audio Record - *Captures audio to a file that can be played back. The file can be deleted.*
+##### Record - *Duration is 60-seconds unless overridden. New duration value is used until it is overridden.*
+> Hey Mycroft, record <br>
+> Hey Mycroft, record audio <br>
+> Hey Mycroft, start recording <br>
+> Hey Mycroft, record me for 6 seconds
+##### Playback 
+> Hey Mycroft, play record <br>
+> Hey Mycroft, play back the audio <br>
+> Hey Mycroft, playback recording
+##### Delete the file
+> Hey Mycroft, delete audio <br>
+> Hey Mycroft, erase recording <br>
+> Hey Mycroft, record clear
+###
+### Configuration - *Interacts with the Device configuration both locally and at home.mycroft.ai.*
+##### Query the registered **name** of this Mycroft Device
+> Hey Mycroft, what is your name?
+##### Query the registered location of this Mycroft Device
+> Hey Mycroft, where are you? <br>
+> Hey Mycroft, tell me your state
+##### Query which listener is being used to detect the Wake Word
+> Hey Mycroft, tell me the wake word listener <br>
+> Hey Mycroft, what is the listener?
+##### Command Mycroft to update the configuation both locally and at home.mycroft.ai	
+> Hey Mycroft, configuration update<br>
+> Hey Mycroft, update config
+##### For developers: Select which listener to use
+> Hey Mycroft, set the listener to precise<br>
+> Hey Mycroft, set the listener to default
+##### For developers: Enable or disable the development version of Precise listener
+> Hey Mycroft, enable the precise dev model <br>
+> Hey Mycroft, disable the new precise model
+###
+### Hello World - *Pleasantries with Mycroft.*
+> Hey Mycroft, greetings <br>
+> Hey Mycroft, hello world
+> 
+> Hey Mycroft, thank you
+> 
+> Hey Mycroft, how are you doing? <br>
+> Hey Mycroft, how was your day?
+
+*Note: Hello World is a well-documented skill that serves as a starting point for skill code development.*
```

Comment:
maybe bump this up under the heading with the description?

Reply:
Sure.  I tried it there initially.  Happy to put it back.